nanopub/client.py

```diff
@@ -214,16 +214,35 @@ def publish(self, nanopub: Publication):
 
         return publication_info
 
-    def claim(self, text, rdftriple=None):
-        """
-        Publishes a claim, either as a plain text statement, or as an rdf triple (or both)
-        """
-        assertion_rdf = rdflib.Graph()
+    def claim(self, statement_text: str, author: str = None):
```

Comment:
I decided to only allow claiming with a statement_text and not with a triple even though that was mentioned in #19 . If you want to publish a triple you can use from_assertion method of Publication. This is just to quickly publish something.